config/development.json

```diff
@@ -148,8 +148,18 @@
 		"perfmon": false,
 		"mailing-lists/unsubscribe": true,
 
-		"jetpack/calypso-first-signup-flow": true
+		"jetpack/calypso-first-signup-flow": true,
+		"sync-handler": true
 	},
+
+	"sync-handler-defaults": {
```

Comment:
Unless this gets copied, we probably need these values in the other config files (desktop, prod, etc).